README.md

```diff
@@ -1,5 +1,49 @@
-Pytest-executable
-=================
+# Pytest-executable
 
-A pytest plugin for testing executables.
-For more informations, please read the [documentation](https://pytest-executable.readthedocs.io/en/latest).
+`pytest-executable` is a [pytest](https://docs.pytest.org/en/latest/) plugin
+for checking and validating executables. This project is still young, but
+already used in a professional environment.
+
+## Documentation
+
+The project documentation is available
+[online](https://pytest-executable.readthedocs.io/en/latest) (or in the
+[`doc`](doc) directory of the repository if you are fluent in
+reStructuredText).
+
+## Installation and Usage
+
+A `pytest-executable` package will soon be available in the Conda and PyPi
```

Comment:
Small typo: conda is the cli tool to interact with anaconda pacakges, they can be hosted in multiple places. The only anaconda repo for providing a community python package is conda-forge.org, we will publish our package there.

Reply:
OK, fixed.